.github/workflows/publish-oss.yml

```diff
@@ -25,14 +25,6 @@ on:
                 description: "Choose the branch to publish from"
                 type: string
                 default: "main"
-            pypi-internal:
```

Comment:
Since we pulled out publishing internally, the only reason to run this workflow is to publish to OSS. Hence neither of these inputs are needed.